source/community/reactnative/src/visualizer/visualizer.js

```diff
@@ -147,9 +162,9 @@ export default class Visualizer extends React.Component {
      * @description Invoked on payload type segmented control status change
      * @param {number} index - index of the selected item
```

Comment:
Update the `@param` based on the changed data type and what it holds.

Reply:
Changes `@param`